### PR TITLE
added Blitzed IRC Trivia

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

File: _data/projects/blitzed-irc-trivia.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+- c
+- c++
+- IRC
+- games
+- trivia
+- cli
+- chat
+upforgrabs:
+  name: Blitzed-Irc-Trivia
+  link: https://github.com/andy5995/blitzed-irc-trivia/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```
Comment (shiftkey) [inline _data/projects/blitzed-irc-trivia.yml]: You can simplify this down to `https://github.com/andy5995/blitzed-irc-trivia/labels/help%20wanted`

**Review by shiftkey — CHANGES_REQUESTED** [review]
Just one little change

File: _data/projects/blitzed-irc-trivia.yml (hunk 1 above)
Comment (andy5995) [reply _data/projects/blitzed-irc-trivia.yml]: Thanks for pointing that out

**Review by shiftkey — APPROVED** [review]

File: _data/projects/blitzed-irc-trivia.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+tags:
+- c
+- c++
+- IRC
+- games
+- trivia
+- cli
+- chat
+upforgrabs:
+  name: Blitzed-Irc-Trivia
```
Comment (shiftkey) [inline _data/projects/blitzed-irc-trivia.yml]: Oops, missed this one. Make this `help wanted`

**Review by shiftkey — CHANGES_REQUESTED** [review]

File: _data/projects/blitzed-irc-trivia.yml (hunk 2 above)
Comment (andy5995) [reply _data/projects/blitzed-irc-trivia.yml]: I wonder if I can have the link point to the "Contribution" guidelines page? From there I link to the issues. The last time I had a project listed here, people would ask how to get started, and I'd always have to point them to my "Contributions" page.

Comment (shiftkey) [reply _data/projects/blitzed-irc-trivia.yml]: @andy5995 Mind opening an issue to discuss this further? Perhaps there's some different fields we can capture here if maintainers want to send users down a different path.

**Review by shiftkey — APPROVED** [review]

File: _data/projects/blitzed-irc-trivia.yml (hunk 2 above)
Comment (andy5995) [reply _data/projects/blitzed-irc-trivia.yml]: Thanks for the merge. No, I don't mind opening up a ticket, thanks for the response.

**Pull request merged**